wayshot/src/cli.rs

```diff
@@ -43,4 +43,8 @@ pub struct Cli {
     /// Present a fuzzy selector for output/display selection
     #[arg(long, alias = "chooseoutput", conflicts_with_all = ["slurp", "output"])]
     pub choose_output: bool,
+
+    /// Uses time stamp(%H:%M:%S) as file name
+    #[arg(short, long)]
+    pub time_stamp: bool,
```

Comment:
```suggestion
    pub timestamp: bool,
```

Timestamp is 1 word, `--timestamp` is also better than `--time-stamp`.